scipy/spatial/transform/_rigid_transformation.pyx

```diff
@@ -9,6 +9,48 @@ cimport cython
 
 np.import_array()
 
+
+def _create_skew_matrix(vec):
+    result = np.zeros((len(vec), 3, 3))
+    result[:, 0, 1] = -vec[:, 2]
+    result[:, 0, 2] = vec[:, 1]
+    result[:, 1, 0] = vec[:, 2]
+    result[:, 1, 2] = -vec[:, 0]
+    result[:, 2, 0] = -vec[:, 1]
+    result[:, 2, 1] = vec[:, 0]
+    return result
+
+
+def _compute_se3_exp_translation_transform(rot_vec):
```

Comment:
Could you add some comments explaining what these methods are doing? They're internal, but take some time to figure out.